### PR TITLE
Add aliases for some functions/modules renamed in Python 3.

### PR DESCRIPTION
**Pull request by luk3yx**
This PR adds the following aliases:
 - `collections.abc` → `_abcoll`
 - ~~`configparser` → `ConfigParser`~~ *(Broken on Windows, this may have to use a meta path importer)*
 - `copyreg` → `copy_reg`
 - `http.client` → `httplib`
 - `http.cookiejar` → `cookielib`
 - `http.cookies` → `Cookie`
 - `http.server` → `BaseHTTPServer`, `CGIHTTPServer`, `SimpleHTTPServer`
 - `importlib.reload` → `reload`
 - `itertools.filterfalse` → `itertools.ifilterfalse`
 - `itertools.zip_longest` → `itertools.izip_longest`
 - ~~`socketserver` → `SocketServer`~~
 - `urllib.error` → `urllib`, `urllib2`
 - `urllib.parse` → `urllib`, `urlparse`
 - `urllib.request` → `urllib`, `urllib2`
 - `urllib.response` → `urllib`
 - `urllib.robotparser` → `robotparser`

### CODE REVIEW

File: Lib/importlib/__init__.py [hunk 1]
```diff
@@ -36,3 +36,5 @@ def import_module(name, package=None):
         name = _resolve_name(name[level:], package, level)
     __import__(name)
     return sys.modules[name]
+
+reload = reload
```
Comment (stefantalpalaru) [inline Lib/importlib/__init__.py]: Why do you need this?

Comment (luk3yx) [reply Lib/importlib/__init__.py]: This adds `importlib.reload`, by default in Python 2 `reload` is a builtin.

File: Lib/urllib/parse.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                    unquote as unquote_to_bytes, urlencode, splitattr,
+                    splithost,
+                    splitpasswd, splitport, splitquery, splittag, splittype,
+                    splituser, splitvalue, unwrap)
+from urlparse import (__doc__, ParseResult, SplitResult, parse_qs, parse_qsl,
+                      urldefrag, urljoin, urlparse, urlsplit, urlunparse,
+                      urlunsplit, uses_fragment, uses_netloc, uses_params,
+                      uses_query, uses_relative)
+
+# Functions modified from Python 3's urllib.
```
Comment (stefantalpalaru) [inline Lib/urllib/parse.py]: How about some tests for these functions?

Comment (luk3yx) [reply Lib/urllib/parse.py]: Added.

**Comment by luk3yx** [comment]
Do I need to add more tests?

**Comment by stefantalpalaru** [comment]
I think we're good. Let me read the changes one more time and I'll merge it.

**Pull request merged**